packages/neutrino-preset-web/index.js

```diff
@@ -67,14 +67,13 @@ module.exports = (neutrino) => {
   neutrino.use(htmlTemplate, neutrino.options.html);
   neutrino.use(namedModules);
   neutrino.use(compileLoader, {
-    include: [neutrino.options.source, neutrino.options.tests],
+    include: [neutrino.options.source, neutrino.options.tests, require.resolve('./polyfills.js')],
```

Comment:
Do we have to add this file to the compilation? Does it work if we switch the polyfills file to use `require`?

```js
require('babel-polyfill');
```